rackcity/models/change_plan.py

```diff
@@ -0,0 +1,22 @@
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class ChangePlan(models.Model):
+    name = models.CharField(max_length=150)
+    execution_time = models.DateTimeField(blank=True, null=True)
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE
```

Comment:
Same comment here about `on_delete=models.CASCADE` - if a user is deleted, their change plan will disappear without warning. I'm not sure that's the desired behavior.

Reply:
this is the expected behavior we settled on, since only users can see their own change plans, also clarified on piazza via this post 
https://piazza.com/class/k4u27qnccr45oo?cid=139